src/compiler/checker.ts

```diff
@@ -48518,29 +48521,6 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return undefined;
     }
 
-    function isLiteralConstDeclaration(node: VariableDeclaration | PropertyDeclaration | PropertySignature | ParameterDeclaration): boolean {

```

Comment:
No longer needed since we only use the syntactic information to determine if we want to preserve the initializer.